packages/general/src/time/Time.ts

```diff
@@ -145,15 +146,24 @@ export class StandardTimer implements Timer {
 
     constructor(
         readonly name: string,
-        readonly intervalMs: number,
+        intervalMs: number,
         private readonly callback: Timer.Callback,
         readonly isPeriodic: boolean,
     ) {
+        this.intervalMs = intervalMs;
+    }
+
+    set intervalMs(intervalMs: number) {
```

Comment:
Please limit set to throw when the timer is running at the moment because then changing it is intransparent